wolf-data/src/Wolf/Data/Export/Types.hs

```diff
@@ -89,3 +105,48 @@ instance ToJSON Repo where
             , "notes" .= repoNotes
             , "suggestions" .= repoSuggestions
             ]
+
+type Warn = [Problem]
+
+data Problem
+    = WarnMissingNoteIndex PersonUuid
+    | WarnMissingNote NoteUuid
+    | WarnMissingRelevantPerson PersonUuid
+                                NoteUuid
+    | WarnMissingRelevantNote NoteUuid
+                              PersonUuid
+    deriving (Show, Eq, Generic)
+
+instance ToJSON Problem
+
+instance FromJSON Problem
+
+data Err =
+    NoInitFile
+    deriving (Show, Eq, Generic)
+
+instance ToJSON Err
+
+instance FromJSON Err
+
+prettyPrintErr :: Err -> String
```

Comment:
It's not pretty printing if you're not printing.

Reply:
https://hackage.haskell.org/package/haskell-src-exts-1.20.2/docs/Language-Haskell-Exts-Pretty.html#v:prettyPrint
That is your interpretation of the word "print".